float32 motor1_control
float32 motor2_control
float32 motor3_control 
